src/platform/site-wide/announcements/sass/style.scss

```diff
@@ -2,6 +2,7 @@
 
 .personalization-announcement {
   position: fixed;
+  z-index: $base-layer + 1;
```

Comment:
This is ultimately `1`, but I used `$base-layer` to be consistent with our z-index approach. Without defining this, the announcement was computing the z-index as `auto`.